templates/locales/en.yml

```diff
@@ -2321,6 +2321,10 @@ en:
           forwarding: Forwarding ports...
           forwarding_entry: |-
             %{guest_port} (guest) => %{host_port} (host) (adapter %{adapter})
+          host_ip_not_found:
+            You are trying to forward a host IP that does not exist. Please set `host_ip`
+            to the address of an existing IPv4 network interface, or remove the option
+            from your port forward configuration.
```

Comment:
It could be useful here to say which guest is setting the option. Sometimes these error messages get lost if the user has a large amount of guests in a Vagrantfile.

Reply:
Side note to that, I'm not sure if the guest name is available where this error is being raised, but if we can make it available I think this will be easier for folks to debug.

Reply:
Thanks for the suggestion, I'll check it out!